templates/contracts/contract-table-template.ejs

```diff
@@ -1,29 +1,34 @@
 <table>
     <th>Network</th>
     <th>Contract address</th>
-
-    <% for (var i = 0; i < rows.length; i++) {%>
+    <% for (let i = 0; i < rows.length; i++) { %>
```

Comment:
It'll be even better if we use `for (const row of rows) {`

Reply:
✅